src/components/TaskTable.vue

```diff
@@ -2,6 +2,10 @@
   <table>
     <thead>
       <tr>
+        <!-- looks very repetitive, how about using a v-for here to abstract this list? -->
```

Comment:
Oops, they had slightly different styling before, so I didn't use the v-for loop. Though, how would I handle them IF they had slightly different styling? For example, there are multiple items that defaults to text-align: center;, and I want to add a utility class(text-align: right;) to the 5th item.

Or would it be the CSS solution, like something like this? 
.someClass:nth-child(5) {
  text-align: right;
}
What if they had slightly different content? like the nth item has an extra little something in the component, like a line of text?

Reply:
use the loop index (I'm not sure how to get it, but when you map or loop over items you can somehow always get the loop index, there should be a simple built-in way to do it) 
and backticks and interpolation, something like this
`text-align: ${idx === 5 ? 'right' : 'center'}`
or `class={idx===5?'textAlignRight':'textAlignCenter'}`

Reply:
for more custom looping, put the attributes in an object, and loop over an array of objects:
`const myListItems = [{ class: 'textAlignCenter', text: "yo I'm the text"},{class: 'textAlignRight', text: "now I'm the text!"}]`

Reply:
`<li v-for="(item, index) in items"> `